app/customapplications.rst

```diff
@@ -8,14 +10,61 @@ from source code.
 Please visit the following sites for working demo setups we have prepared
 for your convenience:
 
-- https://gitlab.com/appuio
-- https://bitbucket.org/appuio
-- https://github.com/appuio
-
-OpenShift Developer Guide
--------------------------
+- `GitLab.com/appuio <https://gitlab.com/appuio?filter=example>`__
+- `Bitbucket.org/appuio <https://bitbucket.org/appuio/?search=example>`__
+- `GitHub.com/appuio <https://github.com/appuio?q=example>`__
 
-Alternatively, you may consult the OpenShift Developer Guide on
-`Creating New Applications
-<https://docs.openshift.com/container-platform/3.11/dev_guide/application_lifecycle/new_app.html>`__,
+Alternatively, you may consult the `OpenShift Developer Guide
+<https://docs.openshift.com/container-platform/3.11/dev_guide/>`__
+on Creating new applications,
 which covers several languages, frameworks and strategies.
+
+Logging
+-------
+
+Every application should log to stdout. APPUiO will automatically collect
+that output and make it accessible via Kibana at `logging.appuio.ch
+<https://logging.appuio.ch>`__. See :ref:`application-logs` for more details.
+
+The Kibana user interface allows powerful queries on all captured application
+console output (you don't have access to cluster logs for security reasons),
+but it requires some data analyst thinking and style of working.
+
+Exception Tracking
+------------------
+
+For APM and exception tracking we suggest to use an external service like
+`Sentry`_, `New Relic`_ or `Datadog`_. (They all provide an API to automate
+configuring your integration, which we highly recommend doing!)
+They all allow to configure alerting or integrate with alert services.
+
+.. _Sentry: https://sentry.io/_/resources/customer-success/alert-rules/
+.. _New Relic: https://docs.newrelic.com/docs/alerts/new-relic-alerts/getting-started/introduction-new-relic-alerts
+.. _Datadog: https://docs.datadoghq.com/monitors/notifications/?tab=is_alert
+
+Monitoring & Alerting
+---------------------
+
+From a concept point of view, cloud applications should be *self-healing* and
+hence do not strictly need to be monitored. You should specify `resource
```

Comment:
Not only requests and limits, but the also the probes should be mentioned, they're much more important.